src/core/Merger.cpp

```diff
@@ -370,7 +371,6 @@ bool Merger::mergeHistory(const Entry* sourceEntry, Entry* targetEntry, Group::M
     }
 
     bool changed = false;
-    const int maxItems = targetEntry->database()->metadata()->historyMaxItems();
```

Comment:
Instead of making this a function parameter, change to `sourceEntry->database()->metadata()->historyMaxItems()`

Reply:
But that would use the history length of the source/alien database in case the branch in line 280 (old) is taken, wouldn't it? That's why I factored out the history length and passed it as parameter.

Reply:
That's a good point, nevermind